SU-HSDO/suhumsci
# feat(STN-129): Carousel Slick Controls Styling for Traditional and Colorful

# READY FOR REVIEW

## Summary

Adds styles for the slick carousel arrows and dots for the traditional and colorful theme

Browser testing
- [x] IE11 - not perfect, but ok:
<img width="1316" alt="Screen Shot 2020-07-13 at 5 25 34 PM" src="https://user-images.githubusercontent.com/12848168/87356369-f4d5ab80-c52f-11ea-9bff-9e5ab2d22856.png">
<img width="1280" alt="Screen Shot 2020-07-13 at 5 39 30 PM" src="https://user-images.githubusercontent.com/12848168/87356386-f99a5f80-c52f-11ea-804e-97b41ae3cb2a.png">


**TODO**
-  ~~test with 2 col, 1 col layout~~

## Steps to Test
1. `npm run test` passes
2. Use the traditional and then colorful theme:
 - add a carousel with 5 slides with varying content to the page (no image, no text, lots of text and image, etc)
 - add a few more carousels in the components section below to test different types of content
 - [x] ensure that the carousel looks good at all breakpoints and the arrows work/always show in the correct place.
  - [x] ensure carousels look ok with a sidebar, and two sidebars layouts

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
